packages/ledger-icp/src/types/ledger.params.ts

```diff
@@ -0,0 +1,9 @@
+import type { QueryParams } from "@dfinity/utils";
+import type { AccountIdentifier } from "../account_identifier";
+import type { AccountIdentifierHex } from "./common";
+
+export type AccountIdentifierParam = AccountIdentifier | AccountIdentifierHex;
```

Comment:
Just for curiosity, why do you need both?

Reply:
The ICP index canister accepts a `String` as input unlike the ICP ledger canister which accepts Unit8Array.
That's why I thought it's more natural to offer both.
In addition, I also thought it's handy to have `String` that way a developper who just want to use the lib to get the account balance does not have to implement the `AccountIdentifier.fromHex` on his/her side and can just call the lib with a string. Like I copy / past an account identifier string somewhere and I can just use it as a param.

Hope that makes sense.

Reply:
Yes, that makes sense. Thanks for the explanation!